prac_06/guitar.py

```diff
@@ -0,0 +1,15 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost:.2f}"
+
+    def get_age(self):
+        guitar_age = 2022 - self.year
```

Comment:
is there a better way to add in current year? possible a constant or class variable